### PR TITLE
Makes Genetics a Bleeding-Edge and Unknown Science again

### PR DESCRIPTION
**Pull request by Tsar-Salat**
## About The Pull Request
Our genetics are a known quantity. Everyone knows what exactly what mutation they got when its activated in their genome. Everyone knows the powergaming genes, and what exactly they are getting when the Geneticist gives it to them. _There is too much Trust._

There is little to no room for sabotage for those with ill-intent for people who supposedly have your entire genetic code at their fingers. People are complacent in powergaming genetics, they know exactly what they are getting.

This PR seeks to reintroduce some of the feeling of the unknown back to genetics through removal of much of the features that give away what genes a person receives.

My changes are as follows:
- Gutted all mutation gain and loss textbox lines in their entirety. No more "You feel Swedish, however that works."

- Fixed the terminology and grammar of a lot of the mutation descriptions to be much more roleplay friendly and immersive. (My example here being my changes to space adapt). 

- A **Genome** is the hereditary palette a carbon starts with or what changes when DNA is scrambled, not a **Mutation**, which is what is gained from the genepod or radiation. I made this terminology consistent across the descriptions.

- Some of the descriptions were really ass. I wanted to make them sound better by adding in some medical terminology and fixing up their grammar

- In addition, I stripped a lot of the genetic instability printlines from the textbox as well. I want this system to work much like Radiation does, in that a lot of the exploration into it needs to be measured and deliberate. A geneticist is going to have to be wary of how many high-powered genes they put into a person, and count the instability, because they will now be sorely regretting it without the reminder that was there before.

## Why It's Good For The Game
The current administration has been pursuing a policy of ensuring better interdepartmental roleplay, especially medbay with the recent tech-lock on health analyzers, but Genetics is dragging behind their department with dated mechanics.

I not only wanted to make Geneticists more important to genes in general, but also to introduce a lot more confusion or uncertainty to Genetics as well. In the world that we are set in, Genetic mutations are supposed to be *Bleeding-Edge* tech, you are literally discovering it as you go.

I feel this is an incredibly easy way to cut down powergaming and make genes an actual threat against the population again. _No rework, no rebalance_. Just cutting out a few words for massive gain.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

nomessage
No message/println appears in the textbox when injecting any mutation

allthesudden
I mutated Three 30 instability genes into myself here, which set off the 90 instability penultimate warning. "You feel your cells burning!" 
Mutating Antenna, a 10 instability gene, set off the final warning.

woopsies!
Woopsies! Too slow!

</details>

## Changelog
:cl: RKz
rebalance: removed all genetic printlines, you can't *feel* Swedish anymore. You better trust that the Genetics Expert gave you something good.
rebalance: removed most genetic stability messages, except for the penultimate and final meltdown one.
spellcheck: Made some mutations descriptions more consistent and immersive. Cleared up terminology misuse.

/:cl:

### CODE REVIEW

**Comment by Tsar-Salat** [comment]
I only did a quick run-through of mutation descriptions. If someone thinks there are others that look ugly and sound weird, tell me and I'll rewrite it.

**Review by Tsar-Salat — COMMENTED** [review]
> /datum/mutation/space_adaptation
> 	name = "Space Adaptation"
> 	desc = "A strange mutation that renders the host's skin, muscuoskeletal system and sensory organs immune to the vacuum of space. The mutation is ineffective toward lung tissue, which will remain vulnerable without an air tank."

This is my favorite desc change

**Comment by PowerfulBacon** [comment]
I don't see how removing player feedback will have a positive impact on genetics

**Comment by EvilDragonfiend** [comment]
There should be a way to check which mutation an injector has, like tapping the injector by gene seq analyzer. I was fine with the injector was null-named, but this will be no way to identify if it's good or bad - even to geneticists.

**Comment by Tsar-Salat** [comment]
> I don't see how removing player feedback will have a positive impact on genetics

Because. Having instant knowledge of whatever mutation you get, either from the genescanner or from radstorms has turned genetics into one of the most easily gameified and oft-powergamed systems in the code.

Virology doesn't instant tell you what you have, Nanites doesn't instant tell you want you have, consuming chemicals doesn't instantly tell you what you consumed.

Simply put, it's a holdover that the other systems in Medical don't share.

**Comment by Tsar-Salat** [comment]
> I was fine with the injector was null-named, but this will be no way to identify if it's good or bad - even to geneticists.

This was sort of what I was going for in this pr. Everyone already knows everything there is about genetics. Stuff like genetic instability or bad genes are just a noobtrap at this point. I mostly want to introduce an element of uncertainty to a system that has basically completely turned into the matching mini game and that's all.


I am willing to change the named injectors though. I am trying to go out of my way to not buff devices like the genetic analyzer, as I personally think people are just going to print them to remove the uncertainty again.

Do you think it will be a problem? If not I will add the names back.

**Comment by PowerfulBacon** [comment]
I guess it can open the path for antagonist geneticists to taint the mutations, however theres not much use you can get out from poisoned mutations.
This will most definitely not solve genetics related powergaming in any capacity.
An alternative, identical in output system, would be to have a chromosome that removes the message from gaining a gene so that you can taint the genes.

Also the removal of the messages that indicate you are about to get qdel'd from too many genes doesn't seem to provide any steps towards making genetics more powergamable, just more confusing.

**Comment by PowerfulBacon** [comment]
> There should be a way to check which mutation an injector has, like tapping the injector by gene seq analyzer. I was fine with the injector was null-named, but this will be no way to identify if it's good or bad - even to geneticists.

You know if its good or bad by if you just printed a good/bad mutator. The name removal change was to avoid mass handing out of genetic syringes since you have no idea what they will do.

**Comment by Tsar-Salat** [comment]
> Also the removal of the messages that indicate you are about to get qdel'd from too many genes doesn't seem to provide any steps towards making genetics more powergamable, just more confusing.

They indicated whether genes activated or not. 

During a radstorm, you might not be sure if you survived without getting a mutation, so I wanted to get rid of the lower level messages. 

This way, while the person would feel the effects of the gene, they also may chalk up their sudden onset nervousness and stuttering to another source, like being too cold or having gotten a disease. 

This patient would then be triaged and diagnosed along with the normal patient population.

Basically, I wanted less beelining towards gene, knowing exactly what you have and how to fix it. 

I can add some of the warnings back in, but I would prefer to keep all the warnings to 50 instability at the least.




> An alternative, identical in output system, would be to have a chromosome that removes the message from gaining a gene so that you can taint the genes.

The traitor perspective wasn't really my primary objective here, it was more of an added bonus.

I put the idea out on ideas-guy that knowing exactly what you received from a line of text was nonsensical to me. My primary motive was cutting that out.

**Comment by Pirill** [comment]
I'm not sure how much this would achieve...genetics is so simple and player-facing as is that removing the messages just seems like an inconvenience. You'd be able to immediately tell if you got injected or irradiated with one of the majority of negative traits even without that message, and likewise most positive ones are immediately obvious through their ability icons and such.

Maybe if the system had a lot more subtlety or complexity to it, it could be an interesting change, but as it is, I'm not convinced.

**Comment by Tsar-Salat** [comment]
> Maybe if the system had a lot more subtlety or complexity to i

I agree. This is ultimately a drop in the pond for what genetics ultimately needs, which is a total rework.

All this intends to do is make genetics more in line for the current vision for medbay. Viro and diagnosis have all developed over the years, yet Gene remains static. 

People immediately know what they have when they get it, and absolutely no nuance is allowed to play out. All I want to do is add a bit of that back.

**Review by PowerfulBacon — APPROVED** [review]

File: code/datums/mutations/sight.dm [hunk 1]
```diff
@@ -36,10 +34,9 @@
 
 /datum/mutation/thermal
 	name = "Thermal Vision"
-	desc = "The user of this genome can visually percieve the unique human thermal signature."
+	desc = "The mutation enables the growth of Heat Pits not unlike those of a reptile, can visually percieve the unique infrared thermal signature of humans."
```
Comment (PowerfulBacon) [inline code/datums/mutations/sight.dm]:
```suggestion
	desc = "The mutation enables the growth of Heat Pits in the eyes, not unlike those of a reptile, which can visually percieve the unique infrared thermal signature of living creatures."
```

File: code/datums/mutations/sight.dm [hunk 2]
```diff
@@ -60,20 +57,18 @@
...
+	desc = "A strange mutation that allows the user to see between the spaces of walls." //actual x-ray would mean you'd constantly be blasting rads, wich might be fun for later //hmb
 	instability = 35
 	locked = TRUE
 	visionflag = TRAIT_XRAY_VISION
 
 //Laser Eyes lets you shoot lasers from your eyes!
 /datum/mutation/laser_eyes
 	name = "Laser Eyes"
-	desc = "Reflects concentrated light back from the eyes."
+	desc = "A mutation that reflects concentrated light back from the eyes at the user's command."
```
Comment (PowerfulBacon) [inline code/datums/mutations/sight.dm]:
```suggestion
	desc = "A mutation that allows for the reflection of concentrated light from the back of the eyes."
```

File: code/datums/mutations/telekinesis.dm [hunk 3]
```diff
@@ -1,10 +1,9 @@
 //Telekinesis lets you interact with objects from range, and gives you a light blue halo around your head.
 /datum/mutation/telekinesis
 	name = "Telekinesis"
-	desc = "A strange mutation that allows the holder to interact with objects through thought."
+	desc = "A strange mutation that allows the holder to interact with objects purely through mental input."
```
Comment (PowerfulBacon) [inline code/datums/mutations/telekinesis.dm]:
```suggestion
	desc = "A strange mutation that allows the holder to interact with objects purely through thought."
```
I think it was more concise before

File: code/datums/mutations/touch.dm [hunk 4]
```diff
@@ -1,11 +1,9 @@
 /datum/mutation/shock
 	name = "Shock Touch"
-	desc = "The affected can channel excess electricity through their hands without shocking themselves, allowing them to shock others."
+	desc = "A mutation that allows the user to store accumulated bioelectric and static charge, and consciously discharge it upon others with no harm to themselves."
```
Comment (PowerfulBacon) [inline code/datums/mutations/touch.dm]:
```suggestion
	desc = "A mutation that allows the user to store accumulated bioelectric and static charge, consciously discharging it upon others with no harm to themselves."
```

**Pull request merged**